### PR TITLE
Suricata: use suricata-update for rule management

### PR DESCRIPTION
**Pull request by adepasquale**
As discussed in #724, use suricata-update for rule management.
- config is in `dist/update.yaml` and `dist/*.conf` (currently only `enable.conf` is used);
- as a bonus we can now run "suricata-update" using docker-exec, triggering both a rule update and a Suricata rule reload.

### CODE REVIEW

File: docker/suricata/Dockerfile.from.source [hunk 1]
```diff
@@ -126,8 +127,6 @@ RUN    apk -U add \
                  nss-dev \
                  nspr-dev \
                  pcre-dev \
-		 python3 \
-                 rust \
```
Comment (adepasquale) [inline docker/suricata/Dockerfile.from.source:130]: I hope this won't increase the `Dockerfile.from.source` image size too much.

File: docker/suricata/Dockerfile [hunk 2]
```diff
@@ -17,13 +17,15 @@ RUN apk -U --no-cache add \
...
+    cp /root/dist/*.conf /etc/suricata/ && \
     cp /root/dist/*.bpf /etc/suricata/ && \
 #
-# Download the latest EmergingThreats ruleset, replace rulebase and enable all rules
+# Download the latest EmergingThreats ruleset
     cp /root/dist/update.sh /usr/bin/ && \
     chmod 755 /usr/bin/update.sh && \
-    update.sh OPEN && \
+    suricata-update update-sources && \
+    suricata-update --no-reload && \
```
Comment (adepasquale) [inline docker/suricata/Dockerfile:28]: There's no need to use the `update.sh` script here anymore.

File: docker/suricata/dist/capture-filter.bpf [hunk 3]
```diff
@@ -1,4 +1,5 @@
 not (host sicherheitstacho.eu or community.sicherheitstacho.eu or listbot.sicherheitstacho.eu) and
+not (host rules.emergingthreats.net or rules.emergingthreatspro.com) and
```
Comment (adepasquale) [inline docker/suricata/dist/capture-filter.bpf:2]: Figured we might add a negation for ruleset download traffic.

File: docker/suricata/dist/enable.conf [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Since honeypot traffic is usually low, we can afford to enable
+# all the rules that are normally disabled for performance reasons.
+re:.
```
Comment (adepasquale) [inline docker/suricata/dist/enable.conf:3]: This is roughly equivalent to `sed "s/^#alert/alert/"`

File: docker/suricata/dist/suricata.yaml [hunk 5]
```diff
@@ -1061,7 +1061,7 @@ host-mode: auto
 # activated in live capture mode. You can use the filename variable to set
 # the file name of the socket.
 unix-command:
-  enabled: no
+  enabled: yes
```
Comment (adepasquale) [inline docker/suricata/dist/suricata.yaml:1064]: We need this in order to issue rule reload via `suricatasc`

**Comment by t3chn0m4g3** [comment]
Shall I prioritize this over #727?

File: docker/suricata/dist/update.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+disable-conf: /etc/suricata/disable.conf
+enable-conf: /etc/suricata/enable.conf
+#drop-conf: /etc/suricata/drop.conf
+modify-conf: /etc/suricata/modify.conf
+
+ignore:
+  - "*deleted.rules"
```
Comment (adepasquale) [inline docker/suricata/dist/update.yaml:7]: Per the discussion in #487, I think this file should be ignored. It's also the default suricata-update behavior to ignore it.

File: docker/suricata/dist/suricata.yaml [hunk 7]
```diff
@@ -1862,78 +1862,15 @@ napatech:
 ## Configure Suricata to load Suricata-Update managed rules.
 ##
 
-#default-rule-path: /var/lib/suricata/rules
-default-rule-path: /etc/suricata/rules
-
+default-rule-path: /var/lib/suricata/rules
 rule-files:
```
Comment (adepasquale) [inline docker/suricata/dist/suricata.yaml:1866]: I did remove this line in a previous commit 🤦

**Comment by t3chn0m4g3** [comment]
LGTM, thank you very much @adepasquale 
Will test the image and publish within the week.

**Pull request merged**